test/mock-interactions.html

```diff
@@ -115,6 +102,7 @@
 
         suite('with touch emulation', function() {
           test('only dispatches one tap event', function() {
+            this.skip();
```

Comment:
The last build run on master was successful https://travis-ci.org/PolymerElements/iron-test-helpers/builds/333046430 - but it was 3 months ago. I'd be curious to know why they're failing, as this seems to be a bug on the dependencies, not on iron-test-helpers.

If we still need to skip these, can we comment here as to why we do it/where is the bug?

Reply:
I run the tests locally and they fail only w/ Polymer 2.x on Chrome
<img width="935" alt="screen shot 2018-05-05 at 10 44 41 am" src="https://user-images.githubusercontent.com/6173664/39666003-87a0c7d4-5051-11e8-8e2e-1e8ed2a08501.png">

Something related with touch emulation
<img width="743" alt="screen shot 2018-05-05 at 10 46 47 am" src="https://user-images.githubusercontent.com/6173664/39666016-af84820e-5051-11e8-8f7d-ad59ea48a7e5.png">

Mind looking into it & skip the test only if really needed?